src/main/java/bogdanSandu/utils/FileUtils.java

```diff
@@ -0,0 +1,20 @@
+package bogdanSandu.utils;
```

Comment:
This class it's also part of homework 6 too. Please remove it from this PR. Thanks!